cluster/v1alpha1/types_rolloutstrategy.go

```diff
@@ -0,0 +1,96 @@
+package v1alpha1
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:deepcopy-gen=true
+
+// RolloutStrategy API used by consumer workload applier APIs such as ManifestWorkReplicaSet, Policy, ManagedClusterAddon ...etc
```

Comment:
we should only indicate that this is for rolling out workload or configuration across clusters. We do not need to mention  ManifestWorkReplicaSet, Policy, ManagedClusterAddon. Otherwise, the doc will be added into each crd definition.